notebooks/examples/python/QuickstartNotebook.py

```diff
@@ -108,26 +146,22 @@
 
 from mosaic import MosaicFrame
 
-mosaicFrame = MosaicFrame(neighbourhoods, "geometry")
-mosaicFrame.get_optimal_resolution(0.75)
+neighbourhoods_mosaic_frame = MosaicFrame(neighbourhoods, "geometry")
+optimal_resolution = neighbourhoods_mosaic_frame.get_optimal_resolution(0.75)
+
+print(f"Optimal resolution is {optimal_resolution}")
 
 # COMMAND ----------
 
 # MAGIC %md
-# MAGIC It is worth noting that not each resolution will yield performance improvements. </br>
-# MAGIC By a rule of thumb it is always better to under index than over index - if not sure select a lower resolution. </br>
+# MAGIC Not every resolution will yield performance improvements. </br>
+# MAGIC By a rule of thumb it is always better to under-index than over-index - if not sure select a lower resolution. </br>
 # MAGIC Higher resolutions are needed when we have very imballanced geometries with respect to their size or with respect to the number of vertices. </br>
 # MAGIC In such case indexing with more indices will considerably increase the parallel nature of the operations. </br>
 # MAGIC You can think of Mosaic as a way to partition an overly complex row into multiple rows that have a balanced amount of computation each.
 
 # COMMAND ----------
 
-display(
```

Comment:
@edurdevic why are we removing this? this was intended to demonstrate why the optimal resolution was selected

Reply:
I thought that this function was removed because I was not able to run this. Maybe the function was renamed?
I would also keep it if you can fix it :)